docs/src/index.md

```diff
@@ -1,5 +1,11 @@
 # RayTracer : Differentiable Ray Tracing in Julia
 
+```@raw html
+<p align="center">
+    <img src="./assets/udem1.gif">
```

Comment:
Isn't this file too large? Almost 30 MB, that's a lot… Maybe use a video? (VP9 with WebM?)